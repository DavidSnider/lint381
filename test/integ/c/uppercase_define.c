#define collection_h

#define COLLECTION_H
