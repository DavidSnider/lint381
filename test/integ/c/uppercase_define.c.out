uppercase_define.c:1:9: error: Macro 'collection_h' should be uppercase
#define collection_h
        ^^^^^^^^^^^^
